docs/content/contributing/bicep/bicep-contribution-flow/_index.md

```diff
@@ -108,11 +108,11 @@ To use the environment's pipelines you should use the information you gathered d
+| `ARM_SUBSCRIPTION_ID` | `22222222-2222-2222-2222-222222222222`                                                                                                                                                                 | The ID of the subscription to test-deploy modules in. Is needed for resources that are deployed to the subscription scope.                                                                                                                                                                 |
+| `ARM_TENANT_ID`       | `33333333-3333-3333-3333-333333333333`                                                                                                                                                                 | The tenant ID of the Azure Active Directory tenant to test-deploy modules in. Is needed for resources that are deployed to the tenant scope.                                                                                                                                               |
 | `AZURE_CREDENTIALS`   | `{"clientId": "44444444-4444-4444-4444-444444444444", "clientSecret": "<placeholder>", "subscriptionId": "22222222-2222-2222-2222-222222222222", "tenantId": "33333333-3333-3333-3333-333333333333" }` | The login credentials of the deployment principal used to log into the target Azure environment to test in. The format is described [here](https://github.com/Azure/login#configure-deployment-credentials). For more information, see the `[Special case: AZURE_CREDENTIALS]` note below. |
-| `TOKEN_NAMEPREFIX`    | `cntso`                                                                                                                                                                                                | Optional. A short (3-5 character length), unique string that should be included in any deployment to Azure. For more information, see the `[Special case: TOKEN_NAMEPREFIX]` note below.                                                                                                   |
+| `TOKEN_NAMEPREFIX`    | `cntso`                                                                                                                                                                                                | Required. A short (3-5 character length), unique string that should be included in any deployment to Azure. Usually, AVM Bicep test cases require this value to ensure no two contributors deploy resources with the same name - which is especially important for resources that require a globally unique name (e.g., Key Vault). For more information, see the `[Special case: TOKEN_NAMEPREFIX]` note below.                                                                                                   |
```

Comment:
I still think this could be elaborated further. Something like "short (3-5 character length), unique string that should be included in any deployment to Azure. These characters will be used as part of the resource's name during deployment."

It still is not clear what the prefix is being used for.

Reply:
Thanks @JFolberth, I added the last bit to the table. 💪

Reply:
@JFolberth if you are good with the change just say the word and I'll give a final approval 🙂

Reply:
cc: @JFolberth careful ping 😉

Reply:
Looks good